packages/scss/src/theming/_breakpoints.scss

```diff
@@ -1,10 +1,11 @@
 $breakpoints: (
-	xs:  576px,
-	sm:  768px,
-	md:  992px,
+	xs: 576px,
+	sm: 768px,
+	md: 992px,
 	lg: 1200px,
 	xl: 1350px,
 	xxl: 1500px,
 	xxxl: 1650px,

```

Comment:
As some tablets are bigger than laptops I prefere a "size relative" naming. 
Furthermore, how should we call biggers breakpoints?